types/react-jsonschema-form/index.d.ts

```diff
@@ -87,6 +88,7 @@ declare module "react-jsonschema-form" {
         formContext: any;
         onBlur: (id: string, value: string) => void;
         onFocus: (id: string, value: string) => void;
+        label: string;
```

Comment:
Had to search for this prop but found it finally on CheckboxWidget. So :+1 from me for this change